### ISSUE TITLE
CGLS data on VITO backend appears empty

### ISSUE THREAD

**Issue by jonathom** [Open-EO/openeo-geopyspark-driver#59]
Although the coverage seems given (see [viewer.terrascope.be](https://viewer.terrascope.be), the CGLS datasets on VITO return empty results. Why is that?

**Comment by jdries** [comment]
CGLS data is not yet fully operational, so some of them work better than others. We have it in the planning to work on those layers. Can you give an indication of which layer is of interest to you and by when?
Thanks!

**Comment by jonathom** [comment]
There's no layer of particular interest to me, I just wanted to point that out in case it was unnoticed / caused by an actual error.

**Comment by jdries** [comment]
I tried a few layers, most of them seem to work, except BA, which I will remove.
There's also this sample:
https://github.com/VITObelgium/notebook-samples/blob/master/datasets/CopernicusGlobalLand/cgls-openEO.ipynb
I also extended integration tests, to ensure that those layers return the expected data.

One important remark is that the resolution of these layers is a lot lower (300m, 1km), so small polygons that work at 10m may  not return data for these layers.
